docs/application/native/guides/device/resource-monitor.md

```diff
@@ -0,0 +1,302 @@
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_CPU_SYS_UTIL`        | double         | Resource attribute for kernel CPU utilization           |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_UTIL`        | array (double) | Resource attribute for per CPU utilization              |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_USER_UTIL`   | array (double) | Resource attribute for per CPU user utilization         |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_SYS_UTIL`    | array (double) | Resource attribute for per CPU kernel utilization       |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_POSSIBLE_CPU`        | int            | Resource attribute for the number of possible CPUs      |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_ONLINE_CPU`          | int            | Resource attribute for the number of online CPUs        |
+
+**Table: Resource attribute keys for Disk**
+
+ | Attribute                                          | Data type | Description                                             |
+ |----------------------------------------------------|-----------|---------------------------------------------------------|
+ | `RESOURCE_MONITOR_DISK_ATTR_NAME`                  | string    | Resource attribute for disk name                        |
+ | `RESOURCE_MONITOR_DISK_ATTR_READ_PER_SEC`          | double    | Resource attribute for read throughput per sec          |
+ | `RESOURCE_MONITOR_DISK_ATTR_WRITE_PER_SEC`         | double    | Resource attribute for write throughput per sec         |
+ | `RESOURCE_MONITOR_DISK_ATTR_READ_TOTAL`            | uint64    | Resource attribute for total read amount                |
+ | `RESOURCE_MONITOR_DISK_ATTR_WRITE_TOTAL`           | uint64    | Resource attribute for total write amount               |
+
+
+ <a name="resource_controls"></a>
+
+The following table lists the required controls for some resources, which are part of the `resource_monitor_ctrl_id_e` enumeration:
+
+**Table: Resource control keys**
+
+ | Control                                            | Resource type | Description                                             |
+ |----------------------------------------------------|---------------|---------------------------------------------------------|
+ | `RESOURCE_MONITOR_CPU_CTRL_CLUSTER_ID`             | CPU           | Resource controls for setting CPU cluster id            |
+ | `RESOURCE_MONITOR_BUS_CTRL_DEVICE_ID`              | BUS           | Resource controls for setting BUS device id             |
+ | `RESOURCE_MONITOR_GPU_CTRL_DEVICE_ID`              | GPU           | Resource controls for setting GPU device id             |
+ | `RESOURCE_MONITOR_DISPLAY_CTRL_DEVICE_ID`          | Display       | Resource controls for setting display device id         |
+ | `RESOURCE_MONITOR_DISK_CTRL_DEVICE_ID`             | Disk          | Resource controls for setting disk device id            |
+
+## Related information
+- Dependencies
+  - Tizen 7.0 and Higher
```

Comment:
@dwlee08,
Please declare the profiles for which this feature will be relevant.

**Example:**
_- Tizen 2.4 and Higher for Mobile_
_- Tizen 3.0 and Higher for Wearable_

Reply:
@omar-srbd, 
Thanks for your suggestion. But, actually our API targeted all profiles, for this case what should we declare for? Is not enough to keep just "Tizen 7.0 and Higher"?

Reply:
@dwlee08 Hello! The dependencies section shows the least supported Tizen version and we need to distinguish how the dependencies are shown for Dot Net vs Native/Web. For Dot Net, the distinct profiles are not a necessity, however, for Native and Web, it should be declared. 
If we are targeting all profiles, I believe it would be prudent to include all of them in the dependencies section. 
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#dependencies